rhamt/widgetastic/__init__.py

```diff
@@ -0,0 +1,66 @@
+from widgetastic.widget import ClickableMixin
+from widgetastic.widget import Text
+from widgetastic.widget import ParametrizedView
+from widgetastic.widget import Widget
+from widgetastic.utils import Parameter
+from widgetastic.utils import ParametrizedLocator
+from widgetastic_patternfly import AggregateStatusCard
+from widgetastic_patternfly import VerticalNavigation
+
+
+class HOMENavigation(VerticalNavigation):
+    """The Patternfly Vertical navigation for Home Icon (Projects)"""
+
+    CURRENTLY_SELECTED = './/li[contains(@class, "home")]/a/span'
+
+
+class RHAMTNavigation(VerticalNavigation):
+    """The Patternfly Vertical navigation for left menu"""
+
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a/span'
+
+
+class ProjectSteps(AggregateStatusCard):
+    ROOT = ParametrizedLocator(".//ul[contains(@class, 'steps-container')]"
+                               "/li[contains(@class, 'steps active')]"
+                               "/span/a[contains(normalize-space(.),{@name|quote})]")
+
+
+class TransformationPath(Widget, ClickableMixin):
+
+    @ParametrizedView.nested
+    class _card(ParametrizedView):
+        PARAMETERS = ("card_name",)
+        card = Text(ParametrizedLocator(
+            './/div[contains(@class, "card-pf-view-multi-select")]'
+            '/div[contains(@class, "card-pf-body")] '
+            "/h2[contains(normalize-space(.),{card_name|quote})]"
+        ))
+
+        def click_card(self):
+            """Clicks the list item with this name."""
+
+            return self.card.click()
+
+    def select_card(self, card_name):
+        return self._card(card_name).click_card()
+
+
+class ProjectList(Widget):
+    ROOT = ParametrizedLocator(".//div[contains(@class,{@list_class|quote})]")
+    list_item_locator = './div[contains(@class,"list-group-item")]'
+    project_locator = ('.//div[contains(@class,"list-group-item-heading")]'
+                       '/a/h2')
+    delete_project = './/span/a[contains(@title, "Delete project")]'
+    edit_project = './/span/a[contains(@class,"Edit Project")]'
```

Comment:
I think we added all those for future enhacent better to have one `TODO` note.

for static locator specification i would use CAPS letters like `ROOT`